learn-stan/tutorials.qmd

```diff
@@ -4,6 +4,39 @@ page-layout: full
 toc-location: left
 ---
 
-# Tutorials
+## Tutorials
+
+* [Getting Started with Bayesian Statistics using Stan and Python](https://github.com/bob-carpenter/stan-getting-started){target="_blank"} &nbsp;Bob Carpenter
+
+* [StanCon2023/Stan_tutorial.ipynb](https://github.com/charlesm93/stanTutorial/blob/main/StanCon2023/Stan_tutorial.ipynb) &nbsp;Charles Margossian
+
+
+## Videos
+
+The official Stan youtube channel is:  [https://www.youtube.com/channel/UCwgN5srGpBH4M-Zc2cAluOA](https://www.youtube.com/channel/UCwgN5srGpBH4M-Zc2cAluOA){target="_blank"}
+
+### Introductory Videos
+
+* [Stan Tutorials](https://www.youtube.com/playlist?list=PLCrWEzJgSUqwL85xIj1wubGdY15C5Gf7H){target="_blank"} &nbsp; Maggie Lieu.
+A series of 8 videos that will get you started using Stan in R and Python.
+
+### Contributed Videos
```

Comment:
How strong do we want to make this distinction?

Reply:
not really necessary.  the YouTube channel has these labels - we don't need to mention them.